docs/examples/java/readme.md

```diff
@@ -0,0 +1,35 @@
+# TDEngine Cloud example
+
+## Introduction
+Use Spring Boot to build a simple web application, and deploy it to connect to TDEngine Cloud.
+
+## Usage
+1. build the project
+```
+mvn clean package -Dmaven.test.skip=true
+```
+2. find the url from cloud page `Programming -> Java -> Config`, and run the project
+```
+java -jar target/example-1.0-SNAPSHOT.jar --spring.datasource.url="jdbc:TAOS-RS://gw.ww-west.pre.cloud.taosdata.com/test?useSSL=true&token=xxxxx"
```

Comment:
please remove the specified URL and add the step how to copy the DSN from Cloud java programming page to this place.